src/components/JobCard/JobCard.scss

```diff
@@ -1,10 +1,10 @@
-.job-card {
-  width: 80%;
-  margin: 8%;
-  background-color: #49a050;
-  color: white;
-  padding: .5em;
-  border-radius: 20px;
-  box-shadow: 4px 4px 10px 1px black;
-  overflow-y: scroll;
-}
+// .job-card {
```

Comment:
Do we no longer need this for some reason?

Reply:
This is actually a duplicate file so we can delete it (I forgot to before pushing my branch up). This file is in the component folder, and the one we are actually pulling the styling from is in the SASS folder.